api/symphony-api-no-k8s.json

```diff
@@ -133,7 +133,7 @@
             "properties": {
               "providers.persistentstate": "k8s-state",
               "singleton": "true",
-              "RetentionInMinutes": "1440"
+              "RetentionInDays": "180"
```

Comment:
I would prefer we can use time string, like "10s, 100m", which provides more freedom for customer to modify. If the customer's value is not correct, we can report error.

Reply:
// ParseDuration parses a duration string.
// A duration string is a possibly signed sequence of
// decimal numbers, each with optional fraction and a unit suffix,
// such as "300ms", "-1.5h" or "2h45m".
// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
func ParseDuration(s string) (Duration, error) {
	// [-+]?([0-9]*(\.[0-9]*)?[a-z]+)+

Reply:
Fixed.